src/ucp/api/ucp_def.h

```diff
@@ -385,13 +385,21 @@ typedef void (*ucp_tag_recv_callback_t)(void *request, ucs_status_t status,
  * @brief UCP worker wakeup events mask.
  *
  * The enumeration allows specifying which events are expected on wakeup, though
- * empty events are possible.
+ * empty events are possible for any type of events excepting @ref UCP_WAKEUP_TX
+ * and UCP_WAKEUP_RX which are the most common events.
+ *
+ * @note UCP doesn't generate POLLOUT-like events (see poll man pages) since
+ * it's always ready to initiate outgoing operations (e.g any type of sending,
+ * atomic or RMA). Hoverer, they can be queued due to limitation of resources.
+ * Send completions are reported by POLLIN-like events.
  */
 typedef enum ucp_wakeup_event_types {
     UCP_WAKEUP_RMA         = UCS_BIT(0), /**< Remote memory access send completion */
     UCP_WAKEUP_AMO         = UCS_BIT(1), /**< Atomic operation send completion */
     UCP_WAKEUP_TAG_SEND    = UCS_BIT(2), /**< Tag send completion  */
     UCP_WAKEUP_TAG_RECV    = UCS_BIT(3), /**< Tag receive completion */
+    UCP_WAKEUP_TX          = UCS_BIT(4), /**< Some data portion was transferred */
```

Comment:
pls elaborate more in the documentation, and define it as UCS_BIT(10), UCS_BIT(11) ( for example ), to separate from the other events

Reply:
May be unite all relevant events? like
`UCP_WAKEUP_TX = UCP_WAKEUP_RMA | UCP_WAKEUP_AMO | UCP_WAKEUP_TAG | UCP_WAKEUP_STREAM_TX` ? and the same for RX